ribs/visualize.py

```diff
@@ -160,7 +163,12 @@ def grid_archive_heatmap(archive,
                       **pcm_kwargs)
 
     # Create the colorbar.
-    ax.figure.colorbar(t, ax=ax, pad=0.1)
+    if cbar == True:
```

Comment:
I don't think we should use `True` here; are there other libraries that do this? It's not very Pythonic to compare directly to True.

Reply:
Maybe we should say that if it is set to "auto" then it puts the default colorbar. Then `cbar="auto"` by default.